src/config.ts

```diff
@@ -119,26 +121,16 @@ export async function configFromCLIOptions(cliOptions: CLIOptions): Promise<Conf
     case 'webkit':
       browserName = 'webkit';
       break;
-    default:
-      browserName = 'chromium';
-      channel = 'chrome';
   }
 
   // Launch options
   const launchOptions: LaunchOptions = {
     channel,
     executablePath: cliOptions.executablePath,
     headless: cliOptions.headless,
+    chromiumSandbox: !!cliOptions.sandbox,
```

Comment:
You just enforced the chromium sandbox by default

Reply:
good catch!